octue/cloud/service_id.py

```diff
@@ -162,3 +163,45 @@ def convert_service_id_to_pub_sub_form(service_id):
         service_id = service_id + "." + service_revision_tag.replace(".", "-")
 
     return service_id
+
+
+def split_service_id(service_id):
+    """Split a service ID into its namespace, name, and, if present, its revision tag.
+
+    :param str service_id: a service ID
+    :return tuple(str, str, str|None): the namespace, name, and revision tag
+    """
```

Comment:
`split_service_id` appears to be able to return a blank string as a revision tag, say if I gave it `octue/service:`. I think that edge case should be handled to always return `None`

Reply:
Good point